composer.json

```diff
@@ -49,6 +49,7 @@
         }
     ],
     "require": {
+        "drupal/layout_builder_usage_reports": "^1.0",
```

Comment:
This needs to go in the `upstream-configuration/composer.json` file instead of the root one.